doc/user/nhrpd.rst

```diff
@@ -180,14 +180,15 @@ Integration with IKE
 nhrpd needs tight integration with IKE daemon for various reasons.
 Currently only strongSwan is supported as IKE daemon.
 
-nhrpd connects to strongSwan using VICI protocol based on UNIX socket
-(hardcoded now as /var/run/charon.vici).
+nhrpd connects to strongSwan using VICI protocol based on UNIX socket which
+can be configured using the command below (default to /var/run/charon.vici).
 
```

Comment:
missing command?

Reply:
I was referring to the  command in the following section. It was added a while ago but this paragraph was not updated